src/main/webapp/WEB-INF/admin/order.jsp

```diff
@@ -0,0 +1,37 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All orders</title>
+</head>
+<body>
+<h1>All orders</h1>
+<table border="1">
+    <tr>
+        <th>User ID</th>
+        <th>Order ID</th>
+        <th>List products</th>
+        <th>Action</th>
+
+        <c:forEach var="order" items="${orders}">
+    <tr>
+        <td>
+            <c:out value="${order.userId}"/>
+        </td>
+        <td>
+            <c:out value="${order.id}"/>
+        </td>
+        <td>
+            <c:out value="${order.products}"/>
+        </td>
```

Comment:
let's add the `details` option here
and use OrderDetailsController and its JSP that will show all products in a table